vsintegration/src/FSharp.Editor/Common/Extensions.fs

```diff
@@ -572,3 +572,24 @@ type Async with
 
             task.Result
         | _ -> Async.RunSynchronously(computation, ?cancellationToken = cancellationToken)
+
+#if !NET7_0_OR_GREATER
+open System.Runtime.CompilerServices
+
+[<Sealed; AbstractClass; Extension>]
+type ReadOnlySpanExtensions =
+    [<Extension>]
+    static member IndexOfAnyExcept(span: ReadOnlySpan<char>, value0: char, value1: char) =
```

Comment:
Honestly it took me a while to get point here :) I think I was confused by the name since `IndexOfAny` has a bit different idea. Not that I can come up with a better name yet though. `IndexOfOther` probably wouldn't help much.

Reply:
Yeah, this and the ones in `ServiceAnalysis.fs` are just (non-vectorized) polyfills for the [APIs added to `System.MemoryExtensions` in .NET 7](https://learn.microsoft.com/en-us/dotnet/api/system.memoryextensions.indexofanyexcept?view=net-7.0). (The names were landed on here: https://github.com/dotnet/runtime/issues/28795#issuecomment-1097040761.)

Reply:
Ohh okay then, let's keep the name.

Reply:
The name notwithstanding, they are super handy methods :)